### ISSUE TITLE
[Wiki] Syntax error 

### ISSUE THREAD

**Issue by VojtechVitek** [CloudyKit/jet#47]
There's a snippet on [wiki page](https://github.com/CloudyKit/jet/wiki):

```go
templateName := "home.jet"
t, err := View.GetTemplate(templateName)
if err != nil {
    // template could not be loaded
}
var w bytes.Buffer // needs to conform to io.Writer interface (like gin's context.Writer for example)
vars := map[string]interface{}
if err = t.Execute(&w, vars, nil); err != nil {
    // error when executing template
}
```

which causes syntax error:
```
cannot use map[string]interface {} literal (type map[string]interface {}) as type jet.VarMap in argument to tpl.Execute
```

Fix:

```diff
-vars := map[string]interface{}
+vars := map[string]reflect.Value{}
```
or 
```diff
-vars := map[string]interface{}
+vars := jet.VarMap{}
```

I wonder -- was it a design change? I kinda like `map[string]interface{}` better than `map[string]reflect.Value`, since I can easily pass my structs as `interface{}` without having to convert them via `reflect.ValueOf()`.

**Comment by jhsx** [comment]
The last one is the correct way, we change this because using map[string]reflect.Value is way faster since there is no need to use reflect.ValueOf on every time we need to evaluate an expression.

**Comment by VojtechVitek** [comment]
Right, that makes sense. Thanks!

**Comment by annismckenzie** [comment]
Thanks for noticing this, I'll fix the wiki pronto! 💃

**Comment by annismckenzie** [comment]
Fixed – closing this.

**Comment by VojtechVitek** [comment]
Thanks 👍

**Comment by VojtechVitek** [comment]
@annismckenzie btw: can you also add example of `vars.Add("name", "value")` as well, so it's clear how to use vars?

**Comment by VojtechVitek** [comment]
btw: Same issue here (linked from main Wiki page): https://github.com/CloudyKit/jet/wiki/Jet-template-syntax#new-in-v2-local-block-variables-with-defaults-and-wrapping-at-invocation-time

**Comment by annismckenzie** [comment]
I will fix those things as well, thanks.

**Comment by annismckenzie** [comment]
Fixed the wiki and added proper `jet.VarMap` usage to the example project (to be merged in via #52) – closing.
